libraries/resource_mysql2_chef_gem.rb

```diff
@@ -8,8 +8,9 @@ class Mysql2ChefGem < Chef::Resource::LWRPBase
       default_action :install
 
       attribute :mysql2_chef_gem_name, kind_of: String, name_attribute: true, required: true
-      attribute :gem_version, kind_of: String, default: '0.3.17'
+      attribute :gem_version, kind_of: String, default: '0.4.4'
       attribute :client_version, kind_of: String, default: nil
+      attribute :package_version, kind_of: String, default: nil
```

Comment:
The `gem_version` property refers to the version of the `mysql2` gem that is installed, `package_version` refers to the APT/yum version of the `mysql-client` package.

Please see the [`mysql_client` resource](https://github.com/chef-cookbooks/mysql/blob/v8.0.4/libraries/mysql_client_installation_package.rb#L17-L23).